src/main/java/my/bookstore/repository/order/OrderRepository.java

```diff
@@ -0,0 +1,18 @@
+package my.bookstore.repository.order;
+
+import java.util.List;
+import my.bookstore.model.Order;
+import my.bookstore.model.User;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+public interface OrderRepository extends JpaRepository<Order, Long> {
+    String getOrdersByUserQuery = "SELECT o FROM Order o JOIN FETCH o.orderItems oi "
+            + "JOIN FETCH o.user WHERE o.user = :user";
+    @Query(getOrdersByUserQuery)
+    List<Order> getAllByUser(Pageable pageable, User user);
+
+    @Query(getOrdersByUserQuery)
+    List<Order> getAllByUser(User user);
```

Comment:
Leave only one JPA method, the second one is redundant. 

you can provide an empty pageable as a parameter.